ui/pages/routes/routes.component.js

```diff
@@ -260,115 +294,120 @@ export default class Routes extends Component {
-        <Authenticated exact path={SNAPS_ROUTE} component={SnapList} />
-        <Authenticated path={SNAPS_VIEW_ROUTE} component={SnapView} />
-        <Authenticated
-          path={`${CONFIRM_TRANSACTION_ROUTE}/:id?`}
-          component={ConfirmTransaction}
-        />
-        <Authenticated path={SEND_ROUTE} component={SendPage} exact />
-        <Authenticated path={SWAPS_ROUTE} component={Swaps} />
-        <Authenticated
-          path={CROSS_CHAIN_SWAP_ROUTE}
-          component={CrossChainSwap}
-        />
-        <Authenticated
-          path={CONFIRM_ADD_SUGGESTED_TOKEN_ROUTE}
-          component={ConfirmAddSuggestedTokenPage}
-          exact
-        />
-        <Authenticated
-          path={CONFIRM_ADD_SUGGESTED_NFT_ROUTE}
-          component={ConfirmAddSuggestedNftPage}
-          exact
-        />
-        <Authenticated
-          path={`${CONFIRMATION_V_NEXT_ROUTE}/:id?`}
-          component={ConfirmationPage}
-        />
-        {
-          ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
-        }
-        <Authenticated
-          path={CUSTODY_ACCOUNT_DONE_ROUTE}
-          component={InstitutionalEntityDonePage}
-          exact
-        />
-        <Authenticated
-          path={INSTITUTIONAL_FEATURES_DONE_ROUTE}
-          component={InstitutionalEntityDonePage}
-          exact
-        />
-        <Authenticated
-          path={CONFIRM_ADD_CUSTODIAN_TOKEN}
-          component={ConfirmAddCustodianToken}
-          exact
-        />
-        <Authenticated
-          path={INTERACTIVE_REPLACEMENT_TOKEN_PAGE}
-          component={InteractiveReplacementTokenPage}
-          exact
-        />
-        <Authenticated
-          path={CONFIRM_ADD_CUSTODIAN_TOKEN}
-          component={ConfirmAddCustodianToken}
-        />
-        <Authenticated
-          path={CUSTODY_ACCOUNT_ROUTE}
-          component={CustodyPage}
-          exact
-        />
-        {
-          ///: END:ONLY_INCLUDE_IF
-        }
-        <Authenticated path={NEW_ACCOUNT_ROUTE} component={CreateAccountPage} />
-        <Authenticated
-          path={`${CONNECT_ROUTE}/:id`}
-          component={PermissionsConnect}
-        />
-        <Authenticated
-          path={`${ASSET_ROUTE}/image/:asset/:id`}
-          component={NftFullImage}
-        />
-
-        <Authenticated path={`${ASSET_ROUTE}/:asset/:id`} component={Asset} />
-        <Authenticated path={`${ASSET_ROUTE}/:asset/`} component={Asset} />
-        <Authenticated
-          path={`${CONNECTIONS}/:origin`}
-          component={Connections}
-        />
-        <Authenticated path={PERMISSIONS} component={PermissionsPage} exact />
-        <Authenticated
-          path={`${REVIEW_PERMISSIONS}/:origin`}
-          component={ReviewPermissions}
-          exact
-        />
-        <Authenticated path={DEFAULT_ROUTE} component={Home} />
-      </Switch>
+      <Suspense fallback={<AppLoadingSpinner />}>
```

Comment:
Has this been approved by the design team, does this match the initial loader on first load?

Reply:
Oh great point, the load is so fast that I didn't even look at it.  But once I slowed it down on purpose, actually you can't even see the spinner.  The extension becomes Header only, and the rest of the page just isn't loaded yet.

Yeah I wonder if it should actually match the orange spinner with the fox, or the gray spinner.  Lemme ask the design team.